### PR TITLE
Additions to pbnify

### PR DESCRIPTION
**Pull request by imouwen**
Hi Dan,

I have made some additions to your code. Hope you like them, thanks.

Regards,
Iwan Mouwen

### CODE REVIEW

**Comment by eric-wieser** [comment]
A lot of these changes look great - I guess you gave up on them due to a lack of response from Dan?

File: scripts/controllers/main.js [hunk 1]
```diff
@@ -43,14 +49,51 @@ angular.module('pbnApp')
...
+	var dist = Math.sqrt((((512+rmean)*r*r)>>8) + 4*g*g + (((767-rmean)*b*b)>>8));
+	return dist;
       };
 
+      $scope.addColor = function(color, threshold) {
+	  addColorInfo(color);
+	  var mindist = -1;
+  	  for (var i = 0; i < $scope.palette.length; i++) {
+               var pcol = $scope.palette[i];
+               if ((color.r == pcol.r) && (color.g == pcol.g) && (color.b = pcol.b)) {
```
Comment (eric-wieser) [inline scripts/controllers/main.js:67]: `==` not `=`

Comment (imouwen) [reply scripts/controllers/main.js:67]: Thanks for spotting this typo. I have fixed it in my repo.

Comment (eric-wieser) [reply scripts/controllers/main.js:67]: I ended up merging your changes into my own fork, and fixed it there too: https://github.com/eric-wieser/pbnify

**Pull request closed without merging**